src/backend/storeManagers/gog/library.ts

```diff
@@ -587,21 +593,37 @@ export async function getMetaResponse(
   etag?: string
 ) {
   const buildData = await axios.get(
-    `https://content-system.gog.com/products/${appName}/os/${platform}/builds?generation=2`
+    `https://content-system.gog.com/products/${appName}/os/${platform}/builds?generation=2&_version=2`
   )
   const headers = etag
     ? {
         'If-None-Match': etag
       }
     : undefined
-  const metaUrl =
-    buildData.data?.items?.find((build: BuildItem) => !build.branch)?.link ||
-    buildData.data?.items[0]?.link
-  const metaResponse = await axios.get(metaUrl, {
-    headers,
-    validateStatus: (status) => status === 200 || status === 304
-  })
-  return { status: metaResponse.status, etag: metaResponse.headers.etag }
+  const metaUrls =
+    buildData.data?.items?.find((build: BuildItem) => !build.branch)?.urls ||
+    buildData.data?.items[0]?.urls
+
+  for (const metaUrl of metaUrls) {
```

Comment:
from the previous line, looks like `metaUrls` can be undefined, maybe add a `[]` as an extra fallback (`... || []`)? or this will fail when undefined

Reply:
I can add a fallback, however there should always be at least one build since this function is called for installed games.

Reply:
oh ok, so maybe there's something wrong in the types? all the `?` makes me think a lot of things there can be undefined
 but from your comment it looks like they can't really be undefined